conjure-java-core/src/integrationInput/java/com/palantir/product/StrictFourFields.java

```diff
@@ -0,0 +1,258 @@
+            missingFields.add(fieldName);
+        }
+        return missingFields;
+    }
+
+    public static MyListStageBuilder builder() {
+        return new DefaultBuilder();
+    }
+
+    public interface MyListStageBuilder {
+        BearerTokenValueStageBuilder myList(@Nonnull List<String> myList);
+
+        Builder from(StrictFourFields other);
+    }
+
+    public interface BearerTokenValueStageBuilder {
+        OptionalItemStageBuilder bearerTokenValue(@Nonnull BearerToken bearerTokenValue);
+    }
+
+    public interface OptionalItemStageBuilder {
+        MappedRidsStageBuilder optionalItem(@Nonnull Optional<String> optionalItem);
+    }
+
+    public interface MappedRidsStageBuilder {
+        Completed_StageBuilder mappedRids(@Nonnull Map<ResourceIdentifier, String> mappedRids);
+    }
+
+    public interface Completed_StageBuilder {
+        StrictFourFields build();
+    }
+
+    public interface Builder
+            extends MyListStageBuilder,
+                    BearerTokenValueStageBuilder,
+                    OptionalItemStageBuilder,
+                    MappedRidsStageBuilder,
+                    Completed_StageBuilder {
+        @Override
+        Builder myList(@Nonnull List<String> myList);
+
+        @Override
+        Builder from(StrictFourFields other);
+
+        @Override
+        Builder bearerTokenValue(@Nonnull BearerToken bearerTokenValue);
+
+        @Override
+        Builder optionalItem(@Nonnull Optional<String> optionalItem);
+
+        @Override
+        Builder mappedRids(@Nonnull Map<ResourceIdentifier, String> mappedRids);
+
+        @Override
+        StrictFourFields build();
+    }
+
+    @Generated("com.palantir.conjure.java.types.BeanBuilderGenerator")
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    static final class DefaultBuilder implements Builder {
+        boolean _buildInvoked;
+
+        private List<String> myList = new ArrayList<>();
+
+        private BearerToken bearerTokenValue;
+
+        private Optional<String> optionalItem = Optional.empty();
+
+        private Map<ResourceIdentifier, String> mappedRids = new LinkedHashMap<>();
+
+        private DefaultBuilder() {}
+
+        @Override
+        public Builder from(StrictFourFields other) {
+            checkNotBuilt();
+            myList(other.getMyList());
+            bearerTokenValue(other.getBearerTokenValue());
+            optionalItem(other.getOptionalItem());
+            mappedRids(other.getMappedRids());
+            return this;
+        }
+
+        @Override
+        @JsonSetter(value = "myList", nulls = Nulls.SKIP)
+        public Builder myList(@Nonnull List<String> myList) {
+            checkNotBuilt();
+            this.myList = ConjureCollections.newArrayList(Preconditions.checkNotNull(myList, "myList cannot be null"));
+            return this;
+        }
+
+        @Override
+        @JsonSetter("bearerTokenValue")
+        public Builder bearerTokenValue(@Nonnull BearerToken bearerTokenValue) {
```

Comment:
Unrelated to this change, but we don't allow `BearerToken`s to have any other safety value than `do-not-log`. Perhaps we should add the annotation here?

Reply:
We don't need to declare log-safety for any type which itself has a type annotation describing log-safety (BearerToken has a `@DoNotLog` annotation declared at the class level)

Reply:
oh, didn't see that. Makes sense.